File: Clients/Java/Add.java
import org.json.simple.JSONArray;
import org.json.simple.JSONObject;

public class Add {
    public static void main(String[] args) {
        PhoenixDB pdb = new PhoenixDB("localhost", 9999, "sudhanshu", "Password@123", "college");

        JSONObject student = new JSONObject();
        student.put("name", "Amit");
        student.put("surname", "Sharma");
        student.put("age", 20);
        student.put("city", "Patna");
        JSONArray skills = new JSONArray();
        skills.add("JAVA");
        skills.add("PHP");
        student.put("skills", skills);

        String response = pdb.add(student.toJSONString(), "students");
        System.out.println(response);

        pdb.close();
    }
}

File: Clients/PHP/add.php
<?php

    print("PhoenixDB<br>");

    require_once('phoenixDB.php');
    $phoenixDB = new PhoenixDB("localhost", 9999, "sudhanshu", "Password@123", "college");
    sleep(1);

    $names = ["Sudhanshu", "Saurabh", "Sparsh", "Rahul"];
    $surnames = ["Jha", "Yadav", "Rathi", "Joshi"];
    $ages = [20, 22, 23, 21];

    for($i=0; $i<sizeof($names); $i++){
        $data = json_encode(
            array(
                'name' => $names[$i],
                'surname' => $surnames[$i],
                'age' => $ages[$i],
                'city' => 'Indore',
                'skills' => ['PHP','C++','JAVA']
            )
        );
        $response = $phoenixDB->add("students", $data);
        print('<pre>');
        print_r($response);
        print('</pre>');
    }

?>